nservicebus/upgrades/host-deprecate.include.md

```diff
@@ -0,0 +1,11 @@
+The following are the main reasons why we are deprecating the NServiceBus Hosts:
```

Comment:
Do we need the plural "hosts" , since its an include perhaps we can just say

>The main reasons for deprecating the host is:

Thoughts?